feature1/one-more.txt

```diff
@@ -0,0 +1,3 @@
+This is also added in the feature-1 branch
```

Comment:
its really good that you have added this line

Reply:
Thanks for this comment